src/commands/emulators-start.ts

```diff
@@ -94,7 +93,13 @@ module.exports = new Command("emulators:start")
         .map((col) => col.slice(0, head.length))
         .filter((v) => v)
     );
-
+    let extensionsTable: string = "";
+    if (EmulatorRegistry.isRunning(Emulators.EXTENSIONS)) {
+      const extensionsEmulatorInstance = EmulatorRegistry.get(
+        Emulators.EXTENSIONS
+      ) as ExtensionsEmulator;
+      extensionsTable = extensionsEmulatorInstance.extensionsInfoTable(options);
+    }
```

Comment:
this lgtm, just some additional thoughts:

printing some post-startup instruction might not be an extension specific issue. wdyt of adding a `getInstruction` method to `EmulatorInstance`, where each `EmulatorInstance` would implement this. Then at the end of `emulators-start` command we can print the post-startup instruction for all running emulators. Could also propose a standardized way to print these instructions that all emulators follow.